src/main.js

```diff
@@ -356,7 +356,7 @@ class MiniGraphCard extends LitElement {
         style="animation-delay: ${this.config.animate ? `${i * 0.5}s` : '0s'}"
         fill='none'
         stroke-dasharray=${this.length[i] || 'none'} stroke-dashoffset=${this.length[i] || 'none'}
-        stroke=${'white'}
+        stroke=#fff
```

Comment:
Need quotes here.

Reply:
Yep, good eye. However, Chrome rendered it just fine as it seems. Also note that before `${'white'}` was also without quotes and worked just fine. Final thought: Should I include it or can we leave it out?

Reply:
Include them.
`${'white'}` should be interpret as a string, hence 'white' I think.
Anyway I'm impressed it worked at all without the quotes 😄

Reply:
The resulting string from `stroke=${'white'}` afaik are just `stroke=white`. But okay, I'll include the 'lil quotes!

Reply:
Yes, you're probably right